extensions/resource-deployment/README.md

```diff
@@ -0,0 +1,17 @@
+# Resource Deployment Extension for Azure Data Studio
+
+Welcome to Resource Deployment Extension for Azure Data Studio!
```

Comment:
>Welcome to [](start = 0, length = 10)

Welcome to the?

Also nit but it would be useful to at least have a one sentence description of what this extension is for. (It's fine though if you're planning on getting a more official description down the line)